introduction.md

```diff
@@ -1,2 +1,5 @@
 Here comes some intro
 JavaScript, of course
+* Git (well, kinda)
+* JS
+* Some other BS
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?